src/main/java/net/fabricmc/loader/launch/knot/MixinContainerHandleMod.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.loader.launch.knot;
+
+import org.spongepowered.asm.launch.platform.container.IContainerHandle;
+
+import java.util.Collection;
+
+public class MixinContainerHandleMod implements IContainerHandle {
+	@Override
+	public String getAttribute(String name) {
+		return null;
+	}
+
+	@Override
+	public Collection<IContainerHandle> getNestedContainers() {
+		return null;
```

Comment:
Should probably return empty collection here, see https://github.com/SpongePowered/Mixin/blob/fd2f2c79c574c0b40340d8977c0e9f284a6e17ed/src/main/java/org/spongepowered/asm/launch/platform/container/ContainerHandleURI.java#L90

Reply:
Good spot